onnx2torch/node_converters/pad.py

```diff
@@ -0,0 +1,109 @@
+__all__ = [
+    'OnnxPadStatic',
+    'OnnxPadDynamic',
+]
+
+from typing import List
+from typing import Optional
+
+import torch
+from torch import nn
+
+from onnx2torch.node_converters.registry import add_converter
+from onnx2torch.onnx_graph import OnnxGraph
+from onnx2torch.onnx_node import OnnxNode
+from onnx2torch.utils.common import OnnxMapping
+from onnx2torch.utils.common import OnnxToTorchModule
+from onnx2torch.utils.common import OperationConverterResult
+from onnx2torch.utils.common import onnx_mapping_from_node
+
+
+def onnx_padding_to_torch(pads: List[int]) -> List[int]:
```

Comment:
Write comment like:
onnx format: [x1b, x2b, ...x1e, x2e, ...] -> torch format [x_n-1_b, x_n-1_e, xb, xe, ...]

Reply:
And make this function 'private' -> `_onnx_padding_to_torch`

Reply:
Done